documentation/manual/src/doc/interpreters/openapi.md

```diff
@@ -6,6 +6,9 @@
 
```

Comment:
The code examples of this page have been extended to include more context. This is how it looks like now:

![Screenshot from 2019-11-25 09-25-37](https://user-images.githubusercontent.com/332812/69524078-968a9700-0f65-11ea-9953-c425c21a4f21.png)

(you can compare with the [current version](http://julienrf.github.io/endpoints/interpreters/openapi.html))